conda_oci_mirror/repo.py

```diff
@@ -1,19 +1,19 @@
 # Packages and functions for them
 
 import datetime
-import distutils.version
 import fnmatch
 import os
 import tarfile
 
+import packaging.version
```

Comment:
We need to add this new dependency to the pyproject.toml file, I think.

Reply:
And also zstandard, now that I see it.

Reply:
Should be fixed now. I think pip comes with `packaging` already, but it's a good idea to mention it explicitly.